client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.modal.html

```diff
@@ -19,7 +19,7 @@ <h2 class="oui-modal__title" data-ng-bind=":: AbbreviatedNumberModal.title"></h2
         <oui-field
             data-label="{{:: 'telephony_abbreviated_numbers_id' | translate }}"
             data-error-messages="{pattern: AbbreviatedNumberModal.pattern.errorMessage}"
-            data-help-text="{{:: 'telephony_abbreviated_numbers_id_help' | translate }}">
+            data-help-text="{{:: 'telephony_abbreviated_numbers_id_help' | translate: {digit:AbbreviatedNumberModal.pattern.startDigit} }}">
```

Comment:
Space after and before brackets : ` {digit:AbbreviatedNumberModal.pattern.startDigit} `

Reply:
This is fixed